ibis/backends/tests/test_uuid.py

```diff
@@ -19,30 +19,25 @@
     'duckdb': "UUID",
     'sqlite': "text",
     'snowflake': 'VARCHAR',
-    'trino': 'varchar(32)' if SQLALCHEMY2 else 'uuid',
+    'trino': 'uuid',
```

Comment:
Can `SQLALCHEMY2` variable be removed now?

Reply:
It looks like not quite yet, due to the single Trino failure on the SQLAlchemy 2.0 build.

Reply:
Fixed!